modelscope/models/cv/self_supervised_depth_completion/dataloaders/calib_cam_to_cam.txt

```diff
@@ -0,0 +1,34 @@
+calib_time: 09-Jan-2012 13:57:47
```

Comment:
this file should be added  to model directory rather than code library, even though this is data releated, which probably on fit for kitti dataset?